### PR TITLE
Add South African health departments

### PR DESCRIPTION
**Pull request by arrival-spring**
PR so that the checks can go through before committing

### CODE REVIEW

File: data/operators/amenity/hospital.json [hunk 1]
```diff
@@ -26,6 +26,118 @@
...
+    {
+      "displayName": "Free State Department of Health",
+      "locationSet": {"include": ["za"]},
+      "matchNames": ["operator/Free State Health Department"],
+      "tags": {
+        "amenity": "hospital",
+        "healthcare": "hospital",
+        "operator": "Q116648781",
+        "operator:type": "government",
+        "operator:wikidata": "Free State Department of Health"
```
Comment (UKChris-osm) [inline data/operators/amenity/hospital.json:66]: Is there a wikidata for "_Free State Department of Health_"?

Comment (arrival-spring) [reply data/operators/amenity/hospital.json:66]: Oops, I'll fix that now

Comment (arrival-spring) [reply data/operators/amenity/hospital.json:66]: Done, but should I have removed the id, as it now looks a bit odd?

Comment (UKChris-osm) [reply data/operators/amenity/hospital.json:66]: The ID's are auto generated so you don't need to worry about those at all 👍

**Pull request merged**